docs/curate-text/process-data/quality-assessment/classifier.md

```diff
@@ -319,6 +317,48 @@ selective_fasttext_filter = FastTextQualityFilter(
 
 :::
 
+:::{tab-item} Two-Stage: Classification + Filtering
+
+You can combine classification with custom filtering:
+
+```python
+from nemo_curator.pipeline import Pipeline
+from nemo_curator.stages.text.io.reader import JsonlReader
+from nemo_curator.stages.text.io.writer import JsonlWriter
+from nemo_curator.stages.text.classifiers import QualityClassifier
+from nemo_curator.stages.text.modules import Filter
+
+# Create pipeline with classification followed by custom filtering
+pipeline = Pipeline(name="classify_then_filter_pipeline")
+
+# Add stages
+read_stage = JsonlReader("input_data/")
+
+# Step 1: Classify documents (adds quality_pred column)
+classify_stage = QualityClassifier(
+    pred_column="quality_pred",
+    model_inference_batch_size=256
+)
+
+# Step 2: Filter based on classification results
+filter_stage = Filter(
+    lambda pred: pred in ["High", "Medium"],  # Keep High and Medium quality
+    filter_field="quality_pred"
+)
```

Comment:
Please remove in favor of just using the `filter_by` parameter in the quality classifier stage.